fold in Bouwe suggestions in EGU25 abstract (#2)
valeriupredoi/EGU2025: VPredoi_Abstract_EGU25_ESSI3-3.txt

simulations to analysis for scientific publications such as the IPCC reports, including reproducing results from
previously published scientific articles as well as allowing scientists to produce new analysis results.

> ESMValTool is FAIR research software because it is: Findable - it is published in community registries, such

Reviewer: Maybe format the four properties of FAIR as a list for improved readability?

Author: I wanted to do exactly that - good call, bud! But then I thought the  abstract may "look" a lot bigger than it is, and was afraid it'd get rejected at submission. It's already edging towards the 500 words limit

Author: FAIR a list in https://github.com/valeriupredoi/EGU2025/pull/2/commits/0b083f3dbe43336c9dcb54f2bb41050559ffcb15

> ESMValTool is FAIR research software because it is: Findable - it is published in community registries, such

Reviewer: ```suggestion
To make ESMValTool a user-friendly community tool suitable for doing open science, it adheres to the FAIR principles for research software. It is: Findable - it is published in community registries, such
```
Maybe add this as an explanation of why we try follow these principles?

as https://research-software-directory.org/software/esmvaltool; Accessible - it can be installed from

> Python package community distribution channels as conda-forge, and the open-source code is available on

Reviewer: ```suggestion
Python package community distribution channels such as conda-forge, and the open-source code is available on
```

Zenodo with a DOI, and on GitHub; Interoperable - it is based on standards: it works with data that follows CF

> conventions and the CMIP data request, its reusable recipes are written in YAML, and provenance is recorded in the

Reviewer: ```suggestion
Conventions and the Coupled Model Intercomparison Project (CMIP) Data Request, its reusable recipes are written in YAML, and provenance is recorded in the
```

W3C PROV format. It supports diagnostics written in a number of programming language, with Python and R being best supported.

> (with source code following the standards and best practices for the respective programming languages); Reusable - it provides

Reviewer: ```suggestion
Its source code follows the standards and best practices for the respective programming languages; Reusable - it provides
```

a well documented recipe format and Python API that allow reusing previous analyses and building new analysis with
previously developed components. Also, the software can be installed from conda-forge and DockerHub and can be tailored by
installing from source from GitHub.

In terms of input data, ESMValTool integrates well with the Earth System Grid Federation (ESGF) infrastructure.
It can find, download and access data from across the federation, and has access to large pools of observational datasets.

> ESMValTool is built around two key scientific software metrics: scalability - ESMValTool

Reviewer: ```suggestion
ESMValTool is built around two key scientific software metrics: scalability and user friendliness. An important aspect of user friendliness is reliability. ESMValTool
```
Maybe add something like this?
